pymbolic/geometric_algebra/__init__.py

```diff
@@ -112,6 +112,11 @@
    :start-after: START_GA_TEST
    :end-before: END_GA_TEST
 
+Usage as an array container
+---------------------------
+
+The following function permits the use of :class:`MultiVector` as a
+:class:`~arraycontext.ArrayContainer`.
```

Comment:
```suggestion
:class:`~arraycontext.ArrayContainer`.

.. autofunction:: register_multivector_as_array_container
```

Reply:
> Why put this here and not in arraycontext? That already mostly depends on pymbolic through loopy.

Unfortunately, you make a lot of sense.

> Is it worth adding a if "arraycontext" in sys.modules: register() to do it automatically on import?

 Behavior that's randomly dependent on import order? Tasty! :P

Reply:
> Behavior that's randomly dependent on import order? Tasty! :P

Haha, fair enough, a bit too magical.